zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/JobHandler.java

```diff
@@ -103,9 +106,16 @@ public void updateEntity(final Record<JobRecordValue> record, final JobEntity en
         .setJobKind(recordValue.getJobKind().name())
         .setFlowNodeId(recordValue.getElementId());
 
+    if (record.getIntent().equals(JobIntent.CREATED)) {
+      entity.setCreatedAt(recordTimestampAsOffsetDateTime);
+      entity.setUpdatedAt(recordTimestampAsOffsetDateTime);
+    } else {
+      entity.setUpdatedAt(recordTimestampAsOffsetDateTime);
```

Comment:
[nit]
looks like this line can be pulled out of both branches of the if-then-else

Reply:
Makes sense